docs/source/tasks/digital_twins/index.md

```diff
@@ -1 +1,32 @@
-# Digital Twins (WIP)
\ No newline at end of file
+# Digital Twins (WIP)
+[asset-badge]: https://img.shields.io/badge/download%20asset-yes-blue.svg
+[reward-badge]: https://img.shields.io/badge/dense%20reward-yes-green.svg
+
+## PushT-v1
```

Comment:
actually for now lets just put this under the table-top tasks section (it is named for two finger grippers, that will be fixed)